types/augments/pixiParticles.d.ts

```diff
@@ -0,0 +1,61 @@
+declare namespace PIXI {
+  namespace particles {
+    type BasicPoint = import('pixi-particles').BasicPoint;
```

Comment:
Not sure how much performance impact the repeated import here has. If it has adverse performance, wouldn't it be better to only do it once somehow?

Reply:
I didn't figure out a way to make it work with only one import. But I can try again.

Reply:
If it isn't possible and especially if it doesn't have a performance impact, it is fine.

Reply:
I found a solution :)